thunder/common.py

```diff
@@ -137,6 +172,17 @@ def last_computation_execution_time(self, /) -> int:
 #   like additional_param_names
 # TODO RC1 Rename this to CompileOptions
 class CompileData:
+    """A class holding data about the compiled object.
+
+    Data include statistics about how it's been called such as ``additional_param_names``.
+
+    Attributes:
+        additional_param_names:
```

Comment:
Let's not have these, they're on their way out.

Reply:
um, like when? really shortly?

Reply:
Sorry, I missed this. They're only used with thunder.compile.